index.html

```diff
@@ -0,0 +1,40 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="style.css">
+    <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/4.7.0/css/font-awesome.min.css">
+    <link href="https://fonts.googleapis.com/css2?family=Poppins:wght@300;500&display=swap" rel="stylesheet">
+    <title>Portofolio</title>
+</head>
+<body>
+    <header>
+        <div class="container">
+        <a class="logo">Saba Ahmad</a>
+          <nav id="items">
+        <i class="fa fa-bars"></i>
+     </nav>
+     </div>
+</header>
+<section>
+<div class="content">
+    <h1>I'm Saba <br> Glad to see <br> you!</h1>
+    <p>
+        I'm a software developer! I can help you build a product , feature or website Look through some of my work and experience! If you like what you see and have a project you need coded, don’t hestiate to contact me.
+    </p>
+</div>
+      <div class="connect">
+        <h2>LET'S CONNECT</h2>
+        <div class="icon">
+      <img src="pics/Vector1.png" alt="">
+       <img src="pics/Linkedin icon.png" alt="">
+      <img src="pics/Vector 3.png" alt="">
+       <img src="pics/Vector 4.png" alt="">
+       <img src="pics/Vector 5.png" alt="">
```

Comment:
kindly use a ul class in order to arrange the images

Reply:
- Good job with this review 👏 . I'd suggest that use give reasons when suggesting changes, this will help the author of the project understand the reasons to make the changes you have suggested 😄 😄 
- also, as I suggested earlier, please try to use emojis when suggesting changes 👍 👍 
- I suppose you mean `ul` tag and not class 🙃 🙃 . please try to fix this

Reply:
The previous reviewer mentioned that you give reasons when suggesting changes, this will help the author of the project understand the reasons to make the changes you have suggested 😄 😄
- #### Kindly update your comment with an appropriate reason for the suggested changes.